include/SMaterial.h

```diff
@@ -386,7 +386,7 @@ namespace video
 

```

Comment:
why havent we nuked this entire file yet?

Reply:
nuked resolved